src/CLI/clitree/cli-xml/mac.xml

```diff
@@ -143,7 +143,7 @@ limitations under the License.
             >
             <PARAM
                 name="address"
-                help="Mac-address of static interface"
+                help="MAC-address of static interface"
```

Comment:
Sometimes the help strings say "MAC address" and other times they say "MAC-address" -- I think we don't need the hyphen. Should be fine to have a hyphen "address-table" though.

Reply:
done